pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/StreamMessageDecoder.java

```diff
@@ -21,7 +21,7 @@
 import com.linkedin.pinot.core.data.GenericRow;
 
 
-public interface KafkaMessageDecoder<T> {
```

Comment:
Do we want to move this to realtime/stream ? it is currently in realtime/impl/kafka

Reply:
yes, good point.